### ISSUE TITLE
yubikey not registered to keepassxc present locks  gnome

### ISSUE THREAD

**Issue by ilippert** [keepassxreboot/keepassxc#11266]
KeePassXC - Version 2.7.9
Revision: 8f6dd13

Qt 5.15.14
Debugging mode is disabled.

Operating system: Fedora Linux 40 (Workstation Edition)
CPU architecture: x86_64
Kernel: linux 6.10.8-200.fc40.x86_64

Enabled extensions:
- Auto-Type
- Browser Integration
- Passkeys
- SSH Agent
- KeeShare
- YubiKey
- Secret Service Integration

Cryptographic libraries:
- Botan 2.19.5

## Overview
setup linux gnome fedora system uses yubikey for authentication, but yubikey is not used for keepassxc.
system is set up to lock screen if yubikey is removed

## Steps to Reproduce
[NOTE]: # ( Provide a simple set of steps to reproduce this bug. )
1. on the setup described above, enter yubikey
2. bring the locked keepassxc window locked in the foreground

result: the system locks

## Expected Behavior
[NOTE]: # ( Tell us what you expected to happen )
the system should not lock.

## Actual Behavior
[NOTE]: # ( Tell us what actually happens )
keepassxc registers the yubikey. and cannot use it. the system locks.

## Context
[NOTE]: # ( Give us any additional information you may have. )

[NOTE]: # ( Pick choices based on your environment )
Operating System: /Linux/
Desktop Env: Gnome/
Windowing System: /Wayland

This is my first issue contribution here. please be patient with me and let me know what other information might be helpful.

**Comment by droidmonkey** [comment]
See https://github.com/keepassxreboot/keepassxc/issues/10077#issuecomment-2047444637

TL;DR, not our bug

**Comment by ilippert** [comment]
Thanks for the link. I am now seeking support in the [Fedora community](https://discussion.fedoraproject.org/t/yubikey-keepassxc-locks-system/139072).
